gmso/external/convert_networkx.py

```diff
@@ -88,4 +88,10 @@ def to_networkx(top):
     for b in top.bonds:
         graph.add_edge(b.connection_members[0], b.connection_members[1], connection=b)
 
+    for node in graph.nodes:
+        graph.nodes[node]['angles'] = top._get_angles_for(node)
+
+    for node in graph.nodes:
+        graph.nodes[node]['dihedrals'] = top._get_dihedrals_for(node)
+
```

Comment:
So if you convert a topology that has bonds, angles, and dihedrals to networkX, you preserve the angle and dihedral info. But if you convert it back to a topology using `from_networkx` you lose that info. Is this the behavior we want or should there be something that parses through the angles and dihedrals in each node and adds them to the topology in `from_networkx`?

Reply:
Good Catch. One way to do this would be to add a flags something like `include_angles` and `include_dihedrals` in this method and add a warning when converting back to topology from nx.Graph.

Reply:
Yeah that's something that was on my mind. It definitely seems like that would be possible. However, I think being able to convert to and from networkx to a topology just isn't a serious way to be playing around with these topology objects. I think this networkx method is a way to take a topology and get some information from it. But I think being able to build usable topologies from a networkx graph just isn't the way we want our users importing information into gmso. So it might just not be worth the time unless someone can suggest a use case that is important to them.

Reply:
Yeah. Not carry extra information that what's needed, I think we can have these node attributes stored conditionally. Basically, these are just references to the original objects inside the topology. So, it might be nice to include them when its absolutely necessary to avoid potential unintended behaviour.

Reply:
However, if we want to include warnings in the `from_networkx` to make sure users know that information is being lost, that would be real quick and easy.

Reply:
Warnings would be fine.

> But I think being able to build usable topologies from a networkx graph just isn't the way we want our users importing information into gmso.

If this is the case, we could just get rid of the `from_networkx` function. [Here is what Justin had to say regarding the use cases for `from_networkx`](https://github.com/mosdef-hub/gmso/pull/472#issuecomment-711308217). But it would be just as easy to force people to only use mbuild for building/modifying connectivity though.

Reply:
I think warnings are completely fine for this case. 

My one argument for keeping from_networkx approximately the same as it is now is that the extra information can be generated on the fly with `identify_connections`. I think for use cases like these, we don’t need to make a lossless conversion necessary. 

Absolutely open to other thoughts on this though.

Reply:
Agree with Justin about keeping this method relatively the same for now. Adding a warning should be sufficient.